docs/4.0/utilities/image-replacement.md

```diff
@@ -18,3 +18,18 @@ Utilize the `.text-hide` class or mixin to help replace an element's text conten
   @include text-hide;
 }
 {% endhighlight %}
+
+The `.text-hide` class is useful for when you want the benefits heading tags like accessibility and SEO, but want to utilize your brand's logo image instead of text.
+
+{% example html %}
+<h1 class="text-hide" style="background-image: url('/assets/brand/bootstrap-solid.svg'); width: 50px; height: 50px;">Bootstrap</h1>
+{% endexample %}
+{% highlight html %}
+<h1 class="text-hide logo">Bootstrap</h1>
+{% endhighlight %}
+{% highlight scss %}
+.logo {
+  background-image: url('/assets/brand/bootstrap-solid.svg');
+}
+{% endhighlight %}
```

Comment:
I'd pick one example here, not a mix of these two as it could confuse beginners. Delete L27-34 maybe?